src/test/java/com/redhat/lightblue/camel/utils/LightblueInsertRequestTransformer.java

```diff
@@ -1,4 +1,4 @@
-package com.redhat.lightblue.camel.transformer;
+package com.redhat.lightblue.camel.utils;
```

Comment:
Save should be enough for all cases. I'm experimenting with LiteralDataRequest now. If that does not work out, I will think about something else.